vnext/Desktop.IntegrationTests/HttpResourceIntegrationTests.cpp

```diff
@@ -17,7 +17,9 @@ TEST_CLASS (HttpResourceIntegrationTest) {
     Assert::IsFalse(nullptr == rc);
   }
 
-  TEST_METHOD(RequestGetSucceeds) {
+  // This test always fails because the requested resource does not exist.
+  // TEST_METHOD(RequestGetSucceeds) {
+  void Disable_RequestGetSucceeds() {
```

Comment:
>Disable_RequestGetSucceeds [](start = 7, length = 26)

remove?

Reply:
Maybe in future. I would like Julio to look at them first.

---
In reply to: [433351796](https://github.com/microsoft/react-native-windows/pull/5071#discussion_r433351796) [](ancestors = 433351796)